### ISSUE TITLE
Where do we put all the amazing examples we can create?

### ISSUE THREAD

**Issue by martell** [google/evcxr#105]
Heyo, does anyone have a good idea about where we could put some cool examples to work with 
```
>> :dep reqwest = "0.10"
>> reqwest::get("https://www.rust-lang.org").await?.status()
200
>>
```

This is my current minimal example showing off `await` and `?` support to make a http request straight from the repl.

I thought it would be nice to have a place where people could collect and show off such things specific to evcxr

**Comment by davidlattimore** [comment]
Good idea. I'd be happy to accept a PR adding an EXAMPLES.md or similar and linking to it from the main README.md.
